plenum/server/node.py

```diff
@@ -2380,9 +2380,21 @@ def doDynamicValidation(self, request: Request):
         State based validation
         """
         self.execute_hook(NodeHooks.PRE_DYNAMIC_VALIDATION, request=request)
+
+        # Digest validation
+        ledger_id, seq_no, digest = self.seqNoDB.get(request.payload_digest)
+        if ledger_id is not None and seq_no is not None and digest is not None:
+            raise SuspiciousPrePrepare('Trying write request with different signatures')
```

Comment:
Maybe a better message is `Trying to order already ordered request`

Reply:
ok